### PR TITLE
Release/storybook pr

### PR DESCRIPTION
**Pull request by zoyar-moveo**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,25 +1,19 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <div>testing test</div>
```
Comment (noam-movhls) [inline src/App.tsx]: drop irrelevant code

File: src/components/Button/Button.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+const Template: Story<ButtonProps> = (args) => <Button {...args} />;
+
+export const WELCOME = Template.bind({});
+WELCOME.args = {
+  text: "text",
+  btnType: "welcome-btn",
+  isArrow: true,
+  onClickFunc: () => {
+    console.log("clicked");
```
Comment (noam-movhls) [inline src/components/Button/Button.stories.tsx]: remove all console.log

File: src/components/Button/Button.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import { ButtonContainer } from "./styles";
+import Arrow from "./assets/arrow-right.svg";
+// import GlobalStyle from "./../../GlobalStyles";
+
+type primary = "primary";
+type secondary = "secondary";
+type textBtn = "text-btn";
+type welcomeBtn = "welcome-btn";
+
+// const btnType = {
```
Comment (noam-movhls) [inline src/components/Button/Button.tsx]: remove

File: src/components/Button/Button.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import { ButtonContainer } from "./styles";
+import Arrow from "./assets/arrow-right.svg";
+// import GlobalStyle from "./../../GlobalStyles";
+
+type primary = "primary";
```
Comment (noam-movhls) [inline src/components/Button/Button.tsx]: check about enums

File: src/components/FeedCard/FeedCard.tsx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    return new Date(publishedAt).toLocaleDateString("en-US", dateOptions);
+  };
+
+  return (
+    <>
+      <GlobalStyle />
+      <CardContainer>
+        <img alt="" src={props.cardImg} />
+        <FlexContainer>
+          <div className="flex row space-between">
```
Comment (noam-movhls) [inline src/components/FeedCard/FeedCard.tsx:48]: better to use styled components

File: src/components/FeedCard/FeedCard.tsx [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+            </CardSecondaryTitle>
+            <TagList text="tag" />
+          </div>
+          <CardHeader>{props.title}</CardHeader>
+          <CardSecondaryTitle>{props.author}</CardSecondaryTitle>
+          <CardBody>{props.content}</CardBody>
+          <div className="flex end">
+            <Button
+              btnType="primary"
+              onClickFunc={() => console.log("clicked")}
```
Comment (noam-movhls) [inline src/components/FeedCard/FeedCard.tsx]: navigation function

File: src/components/TagList/TagList.tsx [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import { TagItem, TagContainer } from "./styles";
+
+export interface TagListProps {
+  text: string;
+}
+
+const TagList: React.FC<TagListProps> = (props) => {
```
Comment (noam-movhls) [inline src/components/TagList/TagList.tsx:7]: Array & how much tag will be hidden

File: src/index.tsx [hunk 8]
```diff
@@ -1,14 +1,16 @@
...
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import reportWebVitals from "./reportWebVitals";
+import GlobalStyles from "./GlobalStyles";
 
 ReactDOM.render(
   <React.StrictMode>
+    <GlobalStyles />
```
Comment (noam-movhls) [inline src/index.tsx]: move to app.tsx

File: src/components/FeedCard/FeedCard.tsx (hunk 5 above)
Comment (zoyar-moveo) [reply src/components/FeedCard/FeedCard.tsx:48]: I'll apply it to the data card and refactor it later here if I'll have time

File: src/components/TagList/TagList.tsx [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+
+export interface TagListProps {
+  tags: string[];
+}
+
+const TagList: React.FC<TagListProps> = (props) => {
+  return (
+    <TagContainer>
+      {props.tags.map((tag, idx) => {
+        return idx < 2 && <TagItem key={idx}>{tag}</TagItem>;
```
Comment (noam-movhls) [inline src/components/TagList/TagList.tsx:11]: magic number

**Review by noam-movhls — APPROVED** [review]

**Pull request merged**